src/schema/rules/datatypes/common_derivatives.yaml

```diff
@@ -0,0 +1,413 @@
+---
+anat_nonparametric_common:
+  $ref: schema.rules.datatypes.anat.nonparametric
+  entities:
+    $ref: schema.rules.datatypes.anat.nonparametric.entities
+    space: OPTIONAL
+    desc: OPTIONAL
+
+anat_parametric_common:
+  $ref: schema.rules.datatypes.anat.parametric
```

Comment:
Since the group has an entities field, will it conflict with the entities defined below when the ref is filled out?

Reply:
The idea is that the below `entities` will override the original. To make it a patch, we pull in the original entities and patch that. So we go from:

```YAML
anat_parametric_common:
  $ref: schema.rules.datatypes.anat.parametric
  entities:
    $ref: schema.rules.datatypes.anat.parametric.entities
    space: OPTIONAL
    desc: OPTIONAL
```

to

```YAML
anat_parametric_common:
  $ref: schema.rules.datatypes.anat.parametric
  entities:
    subject: required
    session: optional
    run: optional
    acquisition: optional
    ceagent: optional
    reconstruction: optional
    space: optional
    desc: optional
```

to

```YAML
anat_parametric_common:
  suffixes:
  - T1map
  - T2map
  - T2starmap
  - R1map
  - R2map
  - R2starmap
  - PDmap
  - MTRmap
  - MTsat
  - UNIT1
  - T1rho
  - MWFmap
  - MTVmap
  - PDT2map
  - Chimap
  - S0map
  - M0map
  extensions:
  - .nii.gz
  - .nii
  - .json
  datatypes:
  - anat
  entities:
    subject: required
    session: optional
    run: optional
    acquisition: optional
    ceagent: optional
    reconstruction: optional
    space: optional
    desc: optional
```